<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(41,49,55,0.85)
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Description">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="text">
     <string>Please Input Your Personal Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="First">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>84</y>
      <width>70</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FirstInput">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>84</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>e.g. Tai Man</string>
    </property>
   </widget>
   <widget class="QLabel" name="Last">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>115</y>
      <width>68</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="text">
     <string>Last Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastInput">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>115</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="placeholderText">
     <string>e.g Chan</string>
    </property>
   </widget>
   <widget class="QLabel" name="User">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>146</y>
      <width>66</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UserInput">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>146</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="placeholderText">
     <string>Enter username</string>
    </property>
   </widget>
   <widget class="QLabel" name="Password">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>177</y>
      <width>64</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordInput">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>177</y>
      <width>125</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF</string>
    </property>
    <property name="placeholderText">
     <string>Enter password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backLoginBtn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>72</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;
border: 1px solid #FFFFFF;
border-radius: 10px</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submitBtn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>210</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #FFFFFF;
border: 1px solid #FFFFFF;
border-radius: 10px</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
